launch/baxter.launch

```diff
@@ -0,0 +1,38 @@
+<launch>
+  <arg name="camera_frame" default="/head_mount_kinect_rgb_optical_frame" />
```

Comment:
We may want to pick better defaults (say, what OpenNI uses).

These should be:
- **camera_frame**: `/camera_rgb_optical_frame`
- **point_cloud_topic**: `/camera/depth_registered/points`
- **depth_topic**: `/camera/depth_registered/image_raw`
- **color_topic**: `/camera/rgb/image_rect_color`

For the simulation, the point cloud topic should remain `/mock_point_cloud`